GoMeet/src/main/webapp/test/spec/spec-vote-meeting-time-dao.js

```diff
@@ -0,0 +1,93 @@
+describe('VoteMeetingTimeDAO - voteMeetingTime', function () {
+  const MEETING_TIME_ID = 'abc123def456';
+  const NON_EXISTENT_ID = 'non-existent-id';
+  const VOTER = 'John Citizen';
+  const QUERY_STRING = 'meetingTimeId=' 
+      + encodeURIComponent(MEETING_TIME_ID) 
+      + '&voters=' 
+      // Note that encodeURIComponenet encodes spaces to %20, 
+      // but URLSearchParam used in VoteMeetingTimeDAO class encodes 
+      // spaces to +. So need to replace that here
+      + encodeURIComponent(VOTER).replace(/%20/g, '+');
```

Comment:
Why is it not replace('%20', '+')

Reply:
Using `replace('%20', '+')` will only replace the '%20' on the first occurrence, but using regex will replace all the %20 in the whole string, which is what I want.

Reply:
But since we are not using user names to identify the user anymore, I've now updated the VOTER to be an email address, in which this replace statement is not used anymore.